### ISSUE TITLE
Photo by ...

### ISSUE THREAD

**Issue by diplomod** [diplomod/nordryggstien.no#137]
We need to be able to indicate the photographer of each photo. It must be entered in submit with the upload of pictures (for each picture).

Then the name should be shown on home (lower right) and in the geofunn details (below the picture thumbnail)

![screen shot 2015-04-12 at 20 15 01](https://cloud.githubusercontent.com/assets/7487883/7106869/ceef3eae-e150-11e4-829a-b2e1b7dded21.png)
![screen shot 2015-04-12 at 20 14 52](https://cloud.githubusercontent.com/assets/7487883/7106870/d3e46628-e150-11e4-8d2c-2e19651ddfbc.png)

**Comment by bokiftw** [comment]
Okay, I can try something like this for slider images, but for geofunn it's a bit complicated to put down there. What do you think about setting it on the hover to show a copyright? Like this: 
![screen shot 2015-04-17 at 10 26 57](https://cloud.githubusercontent.com/assets/5833593/7198542/90f98852-e4ed-11e4-829d-66bf93582e04.png)

**Comment by bokiftw** [comment]
I have done this. You can see here: http://staging.blgrd.co/clients/geonova/nordryggen/wordpress/

Only thing you must take care is when uploading new media, is that you type into the Alt text.
![screen shot 2015-04-22 at 17 11 24](https://cloud.githubusercontent.com/assets/5833593/7277874/a3527382-e912-11e4-91c2-8090ba661822.png)

**Comment by diplomod** [comment]
Very good. I it the same font as in the texts?
And can you make it a little bit bigger? Maybe onl 1 or 2 font size points.

**Comment by diplomod** [comment]
One more request: Can you re-name "Alt Text" to "Photo by"?

**Comment by bokiftw** [comment]
Huh, not really. This is the default and native information about the images and its provided by Wordpress platform. So unfortunately, not.

Anyways, I have made it a little bigger and sent updated files to Lukas.

**Comment by diplomod** [comment]
OK, thanks!

**Comment by diplomod** [comment]
Is it supposed to be working now with the update?

**Comment by bokiftw** [comment]
Yes this should work with this update, but as far as I can see, something is changed here. It's not like I implemented before, that featured image sets the main header image. Also have to see this with Lukas or someone.

**Comment by diplomod** [comment]
This is till not shown. Maybe a problem with the update?
@lukassiedel 
see example:
http://dokuhl.de/nordryggen/geofunn/ura-i-doralglupen/
![screen shot 2015-05-07 at 16 08 04](https://cloud.githubusercontent.com/assets/7487883/7517118/5fe8e5b4-f4d3-11e4-8579-cdba96dfc8e9.png)

![screen shot 2015-05-07 at 16 07 52](https://cloud.githubusercontent.com/assets/7487883/7517110/580dc0b2-f4d3-11e4-8c9d-8dbdb464c8b0.png)

**Comment by diplomod** [comment]
@lukassiedel Bitte das nochmal anschauen, es geht immer noch nicht

**Comment by lukassiedel** [comment]
@bokiftw  i think this should be fixed in the lightbox.js respectively the lightbox.min.js
I can git the alt-text (which should be the author) to the a-tag (with every variablename you want) so you can use it like the title, but you have to include it in the lightbox.js

**Comment by lukassiedel** [comment]
@bokiftw or if you had no more changes to that file i can change it too
